source/usage-examples/changeStream.txt

```diff
@@ -184,3 +166,62 @@ If you run the preceding example, you should see the following output:
 
       const changeStream = collection.watch(pipeline, options);
 
+Listener Function
+~~~~~~~~~~~~~~~~~
+
+The following example opens a change stream on the ``haikus`` collection in
+the ``insertDB`` database. Let's create a listener function to receive and
+print change events that occur on the collection.
+
+First, open the change stream on the collection and then define a listener
+on the change stream using the ``on()`` method. Once set, generate a change
+event by performing a change to the collection.
+
+To generate the change event on the collection, let's use ``insertOne()``
+method to add a new document. Since the ``insertOne()`` may run before the
```

Comment:
```suggestion
To generate the change event on the collection, let's use the ``insertOne()``
method to add a new document. Since ``insertOne()`` may run before the
```